_team/aung.md.orig

```diff
@@ -0,0 +1,12 @@
+---
+firstname: "Amanda"
+lastname: "Ung"
+group: "member"
+graduating_year: 2023
+github: "a17ung"
+title: "Emerson Instructor"
+secondary: "Developer"
+pronouns: "she/her"
+img: "aung.jpg"
+
+---
```

Comment:
Is this file an artefact from the merge conflict? I don't think we should merge it into `master`